nextcord/types/interactions.py

```diff
@@ -202,16 +202,46 @@ class Interaction(TypedDict):
     type: InteractionType
     token: str
     version: int
-    data: NotRequired[InteractionData]
+    # data: NotRequired[InteractionData]
```

Comment:
Because the subclasses recieved their own attribute for data. I did this partially due to typehinting issues, but also because the PingInteraction class (not currently in nextcord, but exists in the dev docs) does not have a data key. (But I mainly did it due to typehint issues - since the `InteractionData` type is a Union of all InteractionData types, and in the various subclasses, pyright would complain in various places, that various code was okay provided if the typehint was the correct one, but would have issues if the type was that used for the other subclasses)

Reply:
Yeah, I forgot about that while reviewing this PR.